src/test/java/org/zalando/nakadi/validation/JSONSchemaValidationTest.java

```diff
@@ -18,105 +17,13 @@
+        ValidationStrategy.register(EventBodyMustRespectSchema.NAME, new EventBodyMustRespectSchema(
+                new JsonSchemaLoader()));
         ValidationStrategy.register(EventMetadataValidationStrategy.NAME, new EventMetadataValidationStrategy());
         ValidationStrategy.register(FieldNameMustBeSet.NAME, new FieldNameMustBeSet());
     }
 
     @Test
-    public void schemaValidationShouldRespectEventTypeDefinition() {
-        final EventType et = EventTypeTestBuilder.builder().name("some-event-type")
-                .schema("{\"type\": \"object\", \"properties\": {\"foo\": {\"type\": \"string\"}, " +
-                        "\"bar\": {\"type\": \"object\", \"properties\": {\"foo\": {\"type\": \"string\"}, " +
-                        "\"bar\": {\"type\": \"string\"}}, \"required\": [\"foo\", \"bar\"]}}, \"required\":" +
-                        " [\"foo\", \"bar\"]}").build();
-
-        final ValidationStrategyConfiguration vsc1 = new ValidationStrategyConfiguration();
-        vsc1.setStrategyName(EventBodyMustRespectSchema.NAME);
-        et.getValidationStrategies().add(vsc1);
-
-        final ValidationStrategyConfiguration vsc2 = new ValidationStrategyConfiguration();
-        vsc2.setStrategyName(FieldNameMustBeSet.NAME);
-        et.getValidationStrategies().add(vsc2);
-
-        final EventTypeValidator validator = new EventTypeValidator(et).withConfiguration(vsc1).withConfiguration(vsc2);
-
-        final JSONObject validEvent = new JSONObject(
-                "{\"foo\": \"bar\", \"bar\": {\"foo\": \"baz\", \"bar\": \"baz\"}, " +
-                        "\"extra\": \"i should be no problem\", \"name\": \"12345\"}");
-        final JSONObject invalidEventMissingBar = new JSONObject(
-                "{\"foo\": \"bar\", \"extra\": \"i should be no problem\", \"name\": \"12345\"}");
-        final JSONObject invalidEventMissingNameField = new JSONObject(
-                "{\"foo\": \"bar\", \"bar\": {\"foo\": \"baz\", \"bar\": \"baz\"}, " +
-                        "\"extra\": \"i should be no problem\"}");
-        final JSONObject nestedSchemaViolation = new JSONObject(
-                "{\"bar\": {\"foobar\": \"baz\"}, \"extra\": \"i should be no problem\", \"name\": \"12345\"}");
-
-        assertThat(validator.validate(validEvent), isAbsent());
-        assertThat(validator.validate(invalidEventMissingBar).get().getMessage(),
-                equalTo("#: required key [bar] not found"));
-        assertThat(validator.validate(invalidEventMissingNameField).get().getMessage(), equalTo("name is required"));
-        assertThat(validator.validate(nestedSchemaViolation).get().getMessage(),
-                equalTo("#: 3 schema violations found\n#/bar: 2 schema violations found\n#/bar: required key [foo]" +
-                        " not found\n#/bar: required key [bar] not found\n#: required key [foo] not found"));
-    }
-
-    @Test
-    public void schemaValidationShouldRespectIgnoreConfigurationMatchRegular() {
```

Comment:
This test no longer makes sense since this code has been finally deleted. It was never used in production.